smithy-typescript-codegen/src/test/resources/software/amazon/smithy/typescript/codegen/test-list-with-union-with-sensitive-data.smithy

```diff
@@ -0,0 +1,25 @@
+namespace smithy.example
+
+service Example {
+    version: "1.0.0",
+    operations: [GetFoo]
+}
+
+operation GetFoo {
+    input: GetFooInput
+}
+
+structure GetFooInput {
+    foo: UserList
+}
+
+list UserList {
+    member: TestUnion
+}
+
+union TestUnion {
+    bar: String,
+
+    @sensitive
+    sensitiveBar: String
+}
```

Comment:
Can you please add newlines to the end of these files? There are quite a few without them.

If you use IntelliJ, you can set this automatically under Preferences ->  Editor -> General -> Ensure an empty line at the end of a file on Save